server/consumer.go

```diff
@@ -988,8 +1019,9 @@ func (o *consumer) setLeader(isLeader bool) {
 		}
 
 		// Update the group on the our starting sequence if we are starting but we skipped some in the stream.
+		// TODO(tp): check how we should approach this for multiple filters. Ask Derek about this one.
```

Comment:
sseq should be proper starting point, so code below should be fine regardless of multiple subjects.

Reply:
Can remove comment.